knowledge_base/Heroku.md

```diff
@@ -1,6 +1,66 @@
 # Heroku
 
-## Production Database
+Engineers should be assigned Heroku admin permissions during the onboarding process. If this is not the case, reach out to an engineering lead.
+
+## Contents
+
+- [Apps](#apps)
+  + [Common-app](#common-app)
+  + [Common-bot](#common-bot)
+- [Procfiles](#procfiles)
+- [Buildpacks](#buildpacks)
+- [Release phases](#release-phases)
+- [Downloading Production Databases](#downloading-production-databases)
+- [Change Log](#change-log)
+
+## Apps
+
+As of 231107, eight apps are hosted on Heroku:
+
+1. `common-app`
+2. `commonbot`
+3. `commonwealth-frack-eu`
+4. `cosmos-devnet`
+5. `cosmos-devnet-beta`
+6. `cw-discourse-import`
+7. `evmos-devnet`
+8. `snapshot-listener`
+
+### Common-app
+
+The `common-app` comprises several sub-apps or servers:
+
+1. `commonwealth-beta`: used in the engineering QA process.
+2. `commonwealth-demo`: used by the Design and Product Teams to validate engineer work and provide feedback before merging deploying it to the livesite.
+3. `commonwealth-experimental`: used to host Common 2.0 development.
+4. `commonwealth-frick`: a full Common instance used for testing bots, chain events, and connected services.
+5. `commonwealth-frack`: a standalone raw Common instance that doesn't interact with other services; it is used for testing pure UI.
+
+The #eng-infra Slack channel is used to communicate which servers are free for testing, and which PRs each sever is presently pegged to.
+
+For more complete information on our testing environments, see the dedicated [Testing Environments](./Testing-Environments.md) entry.
+
+### Common-bot
+
+The common-bot app several sub-apps or servers:
+
+1. `discobot-listener`: Handles Commonwealth <> Discord integration. For full documentation, see [Discobot.md](./Discobot.md).
+2. `discobot-listener-staging`: A testing server for Discobot.
+3. `farcaster-bot-experimental`: An in-progress Common 2.0 project.
+
+## Procfiles
+
+Heroku procfiles are used to specify the processes (workers, dynos, resources) executed by an app on startup. Processes of the "release" type are run during the [release phase](#release-phases). See [dedicated Heroku documentation](https://devcenter.heroku.com/articles/procfile) for a more detailed guide.
```

Comment:
May be worth mentioning here that we have a `multi-procfile` setup (which also means we use a multi-procfile buildpack): https://github.com/heroku/heroku-buildpack-multi-procfile